docs-markdown/src/controllers/lint-config-controller.ts

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+import { ConfigurationTarget, workspace } from "vscode";
+
+// store users markdownlint settings on activation
+let currentMarkdownlintSettings: string[] = [];
+
+export function disableHeaderRule() {
+    // the markdownlint property we need to update
+    const markdownlintProperty = "markdownlint.config";
+    // heading value we need to disable, should append this to the current settings if they exists
+    // may want to use another array for reset
+    const headingFalse = { MD025: false };
```

Comment:
@adunndevster I think we want to keep the MD025 rule active to force users to have a title or H1 tag. But the goal is to allow both title yaml front matter and # H1. Based on this [issue](https://github.com/DavidAnson/markdownlint/issues/199) is looks like this can be done as seen [here](https://github.com/DavidAnson/markdownlint/blob/6f7c0aac131f21dd41a2a26810779c993521b733/test/hugo-quickstart-example-clean.json): 
```
"MD025": {
    "front_matter_title": ""
  }
```
instead of setting MD025 to false.